cmd/controller/app/options/options.go

```diff
@@ -261,6 +271,8 @@ func NewControllerOptions() *ControllerOptions {
 		DefaultIssuerGroup:                defaultTLSACMEIssuerGroup,
 		DefaultAutoCertificateAnnotations: defaultAutoCertificateAnnotations,
 		ACMEHTTP01SolverNameservers:       []string{},
+		ACMEDNS01CheckMethod:              defaultACMEDNS01CheckMethod,
+		DnsOverHttpsJsonEndpoint:          defaultDnsOverHttpsJsonEndpoint,
 		DNS01RecursiveNameservers:         []string{},
 		DNS01RecursiveNameserversOnly:     defaultDNS01RecursiveNameserversOnly,
```

Comment:
`ACMEDNS01CheckMethod`, `DnsOverHttpsJsonEndpoint`, `DNS01RecursiveNameservers`, `DNS01RecursiveNameserversOnly`
After this PR, we will have 4 options, how do we provide our users with a good user experience here?